packages/store/package.json

```
@@ -46,7 +46,7 @@
     "clean:typechain": "rimraf types",
     "dev": "tsup --watch",
     "gas-report": "mud-gas-report --save gas-report.json",
-    "generate-tightcoder": "tsx ./scripts/generate-tightcoder.ts && prettier --write '**/tightcoder/*.sol'",
+    "generate-tightcoder": "tsx ./ts/scripts/generate-tightcoder.ts",
```

should we call this `build:tightcoder` for consistency with the other build scripts? (no strong opinion)
```suggestion
    "build:tightcoder": "tsx ./ts/scripts/generate-tightcoder.ts",
```

---

it's based on `generate-test-tables` from `cli`, I also don't have a strong opinion either way.
`generate-test-tables` and `generate-tightcoder` are different so it's okay for them to diverge too (test tables are for tests only, tightcoder is for everything)
@holic thoughts?

---

changed build:tightcoder in this PR, test-tables in https://github.com/latticexyz/mud/pull/1248